cmd/command/cd/cd_pipelines.go

```
@@ -114,3 +128,26 @@ func (p *Plural) handlePipelineContext(c *cli.Context) error {
 	utils.Success("Pipeline %s context set successfully\n", id)
 	return nil
 }
+
+func (p *Plural) handlePipelineContextFromBlob(c *cli.Context) error {
+	if err := p.InitConsoleClient(consoleToken, consoleURL); err != nil {
+		return err
+	}
+
+	context := c.String("context")
+	if context == "" {
+		return fmt.Errorf("no context provided")
+	}
+
+	raw := json.RawMessage(context)
```

why not use json.Unmarshall?  That way you can return an error